src/utils/gwhisperUtils.cpp

```diff
@@ -1,14 +1,31 @@
 #include "gwhisperUtils.hpp"
 #include <fstream>
+#include <iostream>
+#include <string>
+#include <filesystem>
+
 namespace gwhisper
 {
     namespace util
     {
-        std::string readFromFile(const std::string f_path)
+        std::string readFromFile(std::string f_path)
         {
             std::ifstream credFile;
-            credFile.open(f_path);
 
+            std::size_t foundTilde = f_path.find_first_of("~");
```

Comment:
hm this would not support paths which have tilde in them.
however, we could limit the search to first char only right?